src/searchForm/SearchForm.tsx

```diff
@@ -143,30 +145,11 @@ const SearchForm = ({
     })
   }
 
-  useEffect(() => {
-    function hideAllOverlays(e: Event) {
-      if (!container.current.contains(e.target) && mediaSize === MediaSize.LARGE) {
-        setElementOpened(null)
-      }
-    }
-
-    document.addEventListener('click', hideAllOverlays)
-    return () => {
-      document.removeEventListener('click', hideAllOverlays)
-    }
-  }, [mediaSize])
-
   const autocompleteFromValue = formValues[Elements.AUTOCOMPLETE_FROM] as AutocompleteOnChange
   const autocompleteToValue = formValues[Elements.AUTOCOMPLETE_TO] as AutocompleteOnChange
 
   return (
-    <form
-      action=""
-      noValidate
-      className={cc(['kirk-searchForm', className])}
-      ref={container}
-      role="search"
-    >
+    <form action="" noValidate className={cc(['kirk-searchForm', className])} role="search">
```

Comment:
I think you can remove the `onClick` listener on the button and replace the `action` prop of the form by a `onSubmit={event => { event.preventDefault() }}`.

A button in a form automatically triggers the form submission when clicked so you only need to _catch_ the submit event and prevent the browser's default action.

Reply:
Need to keep `action` so that the keyboard on iOS is the "form keyboard" because there's an issue on forms that don't have as far as I remember. 
But I agree with the other part, I'll move to the onSubmit event instead :+1